packages/git/src/browser/history/git-history-contribution.ts

```diff
@@ -43,6 +46,25 @@ export class GitHistoryContribution extends AbstractViewContribution<GitHistoryW
         });
     }
 
+    @postConstruct()
+    init() {
+        this.repositoryTracker.onDidChangeRepository(async repository => {
+            if (repository && repository.localUri) {
```

Comment:
`localUri` is not optional, it cannot be `undefined`. Why do you check it? Am I missing something?